janitor/functions/change_type.py

```diff
@@ -50,7 +52,7 @@ def change_type(
         2    2   1.0
 
     :param df: A pandas DataFrame.
-    :param column_name: A column in the dataframe.
+    :param column_name: The column(s) in the dataframe.
```

Comment:
How about also adding an example in the docstrings demonstrating that multiple columns are supported?

Because it's not obvious from the parameter name (`column_name`) that this is so